cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -728,15 +768,6 @@ func (p *BicepProvider) purgeAPIManagement(
 	return nil
 }
 
```

Comment:
This is when I learned that ARM automatic deployment history cleanup is now a [thing](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-history-deletions?tabs=azure-powershell)! I'm happy with the approach here.

In the old days, I used to hit deployment limits during prod deployments from time-to-time. Worst timing to hit this is during any high-priority rollouts. We had to write ad-hoc cleanup scripts, which then required privilege elevation for execution -- that was such a mess.